ingress-controllers/readme.adoc

```diff
@@ -250,13 +256,31 @@ your S3 Bucket name for kops configuration and you kops cluster name:
         export KOPS_CLUSTER_NAME=example.cluster.k8s.local
 
 
-The you create the kops cluster and validate that everything is set up properly.
-
+Then you create the kops cluster and validate that everything is set up properly.
 
         export KOPS_STATE_STORE=s3://${S3_BUCKET}
         kops create cluster --name $KOPS_CLUSTER_NAME --zones $AWS_AVAILABILITY_ZONES --cloud-labels kubernetes.io/cluster/$KOPS_CLUSTER_NAME=owned --yes
         kops validate cluster
 
+=== Modify an Existing Cluster
+
+To modify your existing kops cluster and update it.
+
+        export KOPS_STATE_STORE=s3://${S3_BUCKET}
+        kops edit cluster $KOPS_CLUSTER_NAME
+
+Add cloudLabels dependent on your $KOPS_CLUSTER_NAME, for example example.cluster.k8s.local:
+
+```
+ spec:
+   cloudLabels:
+     kubernetes.io/cluster/example.cluster.k8s.local: owned
+```
+
+Update the cluster with the new configuration:
+
+       kops update cluster $KOPS_CLUSTER_NAME --yes
+
```

Comment:
So the update will create the new asg, but after that you need to roll that instance grou with a rolling update command. Someone should test to verify the above statement. How about we get the PR in and test, and then update if we need to.

Reply:
@chrislovecnm It's done late in the IAM part https://github.com/aws-samples/aws-workshop-for-kubernetes/pull/306/files#diff-bc4c670ed3d071dc1009d0d55f868b1dL334

I thought first change everything and then execute the rolling update.

Reply:
Agreed